### ISSUE TITLE
Support Cassandra 2.0.x

### ISSUE THREAD

**Issue by java8964** [fullcontact/hadoop-sstable#23]
I have one dataset already upgraded to Cassandra 2.0. I am trying to see if "hadoop-sstable" can be changed to support Cassandra 2.0.

One problem I found out is that in Cassandra 2.0, for the class SSTableIdentityIterator, Cassandra changed the constructor of taking "DataInput" to a private constructor, see this link:

https://github.com/apache/cassandra/blob/cassandra-2.0.10/src/java/org/apache/cassandra/io/sstable/SSTableIdentityIterator.java

On line 81.

This brings a problem that it is hard to create a SSTableIdentityIterator instance, as the public constructors only take Cassandra RandomAccessReader, not fullcontact's RandomAccessReader.

I wonder what could be a good solution in this case? Do you guys plan anything for supporting Cassandra 2.x?

Thanks

**Comment by bvanberg** [comment]
Yes, and we've done it already. Let me see if our internal 2.0 support can be pushed today.

**Comment by java8964** [comment]
Great, I am interested to test it with our data.

Thanks

**Comment by bvanberg** [comment]
Please try the cassandra-2.0.x branch.
